draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1218,568 +1044,686 @@ uCDN-A                         dCDN-B                         dCDN-C
+                 before comparison with the given regex. This makes the regular expression simpler
+      and safer for cases in which the query parameters are not relevant for the match.
+   </t>
 
-
-   <t hangText="Name: generic-trigger-spec-value">
+        <t hangText="Name: url-type">
         <vspace blankLines="1"/>
-        Description: A CDNI Trigger spec object.
+        Description: Type of URLs to match against.
         <vspace blankLines="1"/>
-   Value: Defined by the value of the generic-trigger-spec-type property.
+        Value: URL Type as defined in <xref target="url-types"/>.
         <vspace blankLines="1"/>
-        Mandatory: Yes.
-        </t>
-   </list></t>
+   Mandatory: No. When omitted or empty, "published" URL type is assumed.
+   </t>
+   </list></t> 
    </list></t>
 
-        <t>The structure of a JSON serialized GenericTriggerSpec object, containing a specific trigger spec is illustrated below:</t>
+   <t>Example of a case-sensitive, no query parameters, regex match against is below.</t>
+   <t>Please note that some lines in the example are wrapped for clarity.</t>
+   <sourcecode type="json" name="regex-match-no-query-param">
+<![CDATA[
+"^(https:\/\/video\.example\.com)\/([a-z])\/
+ movie1\/([1-7])\/*(index.m3u8|\d{3}.ts)$"
+   ]]></sourcecode>
 
-      <sourcecode type="json" name="GenericTrigger Spec">
-             <![CDATA[ 
+            <sourcecode type="json" name="regex-match-2"><![CDATA[
 {
-  "generic-trigger-spec-type":
-     <Type of this trigger spec>,
-  "generic-trigger-spec-value":
-      {
-        <properties of this trigger spec object>
-      },
-   "generic-trigger-spec-subject":
-     <Category of this trigger spec subject>
-
+  "regex": "^(https:\\/\\/video\\.example\\.com)\\/([a-z])\\/
+           movie1\\/([1-7])\\/*(index.m3u8|\\d{3}.ts)$",
+  "case-sensitive": true,
+  "match-query-string": false
 }
-       ]]></sourcecode>
-          </section>
-
+         ]]></sourcecode>
 
+          <t>This regex matches URLs of the domain "video.example.com" where the path structure is
+             /(single lower case letter)/(name-of-title)/(single digit between 1 to 7)/(index.m3u8 or a 3 digit number with ts extension).
+             For example:</t>
+     <sourcecode type="json" name="regex-match-3"><![CDATA[ https://video.example.com/d/movie1/5/index.m3u8 ]]></sourcecode>
+     <t>or</t>
+     <sourcecode type="json" name="regex-match-3"><![CDATA[ https://video.example.com/k/movie1/4/013.ts ]]></sourcecode>
         </section>
-        <section anchor="trigger-extensions" title="CI/T Trigger Extensions">
+   </section>
+
+
+      <section anchor="objectlist-spec" title="Object List Spec">
+      <t>
+        The "objectlist" spec type allows uCDN to manage content held by dCDN based on structured object lists. 
+        The Object List spec type is valid only for the content spec subject (see <xref target="trigger-subject"/>).
```

Comment:
Is there a reason for this restriction?  Is there any reason why it would not be good to refer to a set of metadata URLs using an Object List?

Reply:
No, arrived to the same conclusion when I was putting together the spec constraints matrix. Fixed here as well.